apollo-router/tests/snapshots/integration_tests__traced_basic_composition.snap

```diff
@@ -399,7 +394,7 @@ expression: get_spans()
                   "record": {
                     "entries": [
                       [
-                        "apollo_private.path",
+                        "path",
```

Comment:
Yes path is not something that appears in our traces.

Reply:
Should we namespace it to be more otel specs compliant ?

Reply:
Yes, Will make the change

Reply:
Fixed in [38b5983](https://github.com/apollographql/router/pull/2190/commits/38b59835e2b322ecf9ac74af0cdcf010eb32a08c)